### PR TITLE
Added blend parameter for setPedAnimation

### PR DESCRIPTION
**Pull request by lex128**

### CODE REVIEW

**Comment by qaisjp** [comment]
Please can you modify `CStaticFunctionDefinitions::SetPedAnimation` to conform to the same syntax as `Ped.RunNamedAnimation`

Also, please revert "Fix compatibility" (this would be added in 1.6)

File: Client/mods/deathmatch/logic/CClientPed.cpp [hunk 1]
```diff
@@ -2737,7 +2737,7 @@ void CClientPed::StreamedInPulse ( bool bDoStandardPulses )
                 // Copy our name incase it gets deleted
                 SString strAnimName = m_strAnimationName;
                 // Run our animation
-                RunNamedAnimation ( m_pAnimationBlock, strAnimName, m_iTimeAnimation, m_bLoopAnimation, m_bUpdatePositionAnimation, m_bInterruptableAnimation, m_bFreezeLastFrameAnimation );
```
Comment (qaisjp) [inline Client/mods/deathmatch/logic/CClientPed.cpp:2740]: Shouldn't `0` actually be `250`?

Comment (lex128) [reply Client/mods/deathmatch/logic/CClientPed.cpp:2740]: `0` for faster applying animation after streaming
Or still need smooth?

Comment (qaisjp) [reply Client/mods/deathmatch/logic/CClientPed.cpp:2740]: Hmm, you've got a good point. This'll need some extra testing to make sure this is what we actually want.

**Comment by qaisjp** [comment]
Making a final change for ~~script backwards compatibility~~ the APi, but nice job :)

**Comment by lex128** [comment]
But script backward compatibility is already done by the condition `argStream.NextIsNumber` :wink:
And I think it would be more logical write time `iBlend` right after time `iTime`
Work both variants:

``` lua
bool setPedAnimation ( ped thePed [, string block=nil, string anim=nil, int time=-1, bool loop=true, bool updatePosition=true, bool interruptable=true, bool freezeLastFrame = true] )
bool setPedAnimation ( ped thePed [, string block=nil, string anim=nil, int time=-1, int blend=250, bool loop=true, bool updatePosition=true, bool interruptable=true, bool freezeLastFrame = true] )
```

In any case, this is not so important)

**Pull request merged**